Providers/DB_ServiceProvider.php

```diff
@@ -157,9 +157,12 @@ public function registerViews()
 
         $this->publishes([$sourcePath => $viewPath,], 'views');
 
-        $this->loadViewsFrom(array_merge(array_map(function ($path) {
-            return $path . '/modules/DisposableBasic';
-        }, \Config::get('view.paths')), [$sourcePath]), 'DBasic');
+        $this->loadViewsFrom(array_map(function ($path) {
```

Comment:
```suggestion
        $this->loadViewsFrom(array_map(function ($path) use ($sourcePath) {
```
Will solve the undefined variable exception